src/fileHelper.ts

```diff
@@ -0,0 +1,46 @@
+import * as fs from 'fs'
+
+let containerScanDirectory = '';
+
+export function getFileJson(path: string): any {
+    try {
+        const rawContent = fs.readFileSync(path, 'utf-8');
+        const json = JSON.parse(rawContent);
+        return json;
+    } catch(ex) {
+        throw new Error(`An error occured while parsing the contents of the file: ${path}. Error: ${ex}`);
+    }
+}
+
+export function getContainerScanDirectory(): string {
+    if(!containerScanDirectory) {
+        containerScanDirectory = `${process.env['GITHUB_WORKSPACE']}/_temp/containerscan_${Date.now()}`;
+        ensureDirExists(containerScanDirectory);
+    }
```

Comment:
Use a different convention for global variables maybe?

Reply:
Changing to `CONTAINER_SCAN_DIRECTORY`